### ISSUE TITLE
Document: How to add a shim/tool that's not in bio.tools?

### ISSUE THREAD

**Issue by vedran-kasalica** [Workflomics/workflomics-readthedocs#15]
Provide guidelines on how to add a tool that's not in bio.tools.

An example is [protXml2IdList](https://github.com/Workflomics/tools-and-domains/tree/main/cwl-tools/protXml2IdList). Ideally the tool should be added to bio.tools, but if it is a tool-specific shim, the tool can be added as a shim to the domain without the need for it to be in bio.tools.

**Comment by magnuspalmblad** [comment]
This is one of the shims on [shic](https://github.com/magnuspalmblad/shic) - in fact the first one in the list!

**Comment by vedran-kasalica** [comment]
Yes, I forgot about that. I guess we won't have this example for long then 😅 but we can point to a specific version of the repo that includes [protXml2IdList](https://github.com/Workflomics/tools-and-domains/tree/main/cwl-tools/protXml2IdList). In addition, we should reference SHIC in the documentation as well! Both examples would be useful to domain experts/annotators.
